framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1483,21 +1495,38 @@ private Object getValueOfExpressionInAnnotation(
      * To properly build @Anno, the IntegerLiteralExpr "1" must be converted from an int to a long.
      * */
     private Object convert(Number number, TypeKind expectedKind) {
+        return convert(number, expectedKind, false);
+    }
+
+    /**
+     * Converts {@code number} to {@code expectedKind}. The value converted is multiplied by -1 if
+     * {@code negate} is true
+     *
+     * @param number Number value to be converted
+     * @param expectedKind converts the Number object to one of type {byte, short, int, long, char,
```

Comment:
This is not documentation of the parameter.  It doesn't say anything about `expectedKind`.
Do you want to delete the first 5 words of the parameter documentation, or do something else?